packages/core/src/stepFunctions/activation.ts

```diff
@@ -124,6 +125,19 @@ export function initalizeWebviewPaths(
     }
 }
 
+async function startLspServer(extensionContext: vscode.ExtensionContext) {
+    const perflog = new PerfLog('stepFunctions: start LSP client/server')
+    await ASLLanguageClient.create(extensionContext)
+    perflog.done()
+}
+
+function editingASLFromPrevSession() {
```

Comment:
would it make sense for this to be isASLFileOpen? (Or something like that?) The fact that its from a previous session shouldn't really matter. Technically the only thing that matters is if its open during the execution of the flow

Reply:
Yeah sure I can make that change, I think `isASLFileOpen()` makes sense.